EasyLearn/app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<section class="vh-100 gradient-custom">
+	<div class="container h-100">
+		<div class="row d-flex justify-content-center align-items-center h-100">
+			<div class="col-12 col-md-8 col-lg-6 col-xl-5">
+				<div class="card bg-dark text-white login_card" style="border-radius: 1rem;">
+					<div class="card-body p-5 text-center">
+						<div class="w-100">
+							<h2 class="login_card_title" >Easy Learn</h2>
+							<%= image_tag("easylearn-icon.png", alt: "rss feed", class: "login_img") %>
+							<div class="text-divider">Login</div>
+							<%= form_with(scope: :session, url: login_path) do |f| %>
+								<div class="form-group">
+									<%= f.label :email, class: "d-flex justify-content-start login_label" %><br/>
+									<%= f.email_field :email, class: "form-control login_input"  %>
+								</div>
+								
+								<div class="form-group">
+									<%= f.label :password, class: "d-flex justify-content-start login_label" %><br/>
+									<%= f.password_field :password, class: "form-control login_input" %>
+								</div>
+								
+								<div class="form-group">
+									<%= f.submit "Log in", class: "btn btn-dark login_button mb-3" %>
+								</div>
+								
+								<% end %>	
+							<hr>
+							<p >Or create a new account <%= link_to "here",signup_path, class: "fw-bold login_a" %></p>
+              
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+</section>
```

Comment:
Check the identation tabs, should be 2 spaces

Reply:
This identation is correct, but idk why git hub show it of this way

Reply:
I agree with @Anisbeflo

Reply:
@JuanequeX Then they need to use this in order to fix it https://www.10bestdesign.com/dirtymarkup/css/

Reply:
done